App/Components/GroupCard/index.tsx

```diff
@@ -63,8 +65,8 @@ const GroupCard = (props: ScreenProps) => {
       </View>
       <View style={styles.groupRightColumn}>
         <View style={styles.avatarContainer}>
-          { members.slice(0, 8).map((mem: pb.IContact, i: number) => {
-            const imageStyle = cardImageStyle(members.length, i)
+          { memberArray.slice(0, 8).map((mem: pb.IContact, i: number) => {
```

Comment:
This was a hidden bug I guess... will open a ticket to investigate the source type being labeled incorrectly as an array